File: lang/python/gusu.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-


def gusu(start, end):
    list1 = []
    for num in range(start, end):
        if num % 2 == 0:
            list1.append(num)

    return list1


print gusu(1, 100)

File: lang/lua/gusu.lua
#!/usr/bin/env luajit


function gusu(min, max)
    array = {}
    for i = 1, max + 1 do
        if i % 2 == 0 then
            table.insert(array, i)
        end
    end

    return array
end

array = gusu(1, 100)
num = #array
for i = 1, num do
    print(i .. ": " .. array[i])
end
